beacon-chain/p2p/subnets.go

```diff
@@ -221,6 +235,46 @@ func computeSubscribedSubnets(nodeID enode.ID, epoch primitives.Epoch) ([]uint64
 	return subs, nil
 }
 
+func computeCustodyColumns(nodeID enode.ID) ([]uint64, error) {
+	subs, err := computeSubscribedColumnSubnets(nodeID)
+	if err != nil {
+		return nil, err
+	}
+	colsPerSub := params.BeaconConfig().NumberOfColumns / params.BeaconConfig().DataColumnSidecarSubnetCount
+	colIdxs := []uint64{}
+	for _, sub := range subs {
+		for i := uint64(0); i < colsPerSub; i++ {
+			colId := params.BeaconConfig().DataColumnSidecarSubnetCount*i + sub
+			colIdxs = append(colIdxs, colId)
+		}
+	}
+	return colIdxs, nil
+}
+
+func computeSubscribedColumnSubnets(nodeID enode.ID) ([]uint64, error) {
```

Comment:
Actually I already implemented this [here](https://github.com/prysmaticlabs/prysm/pull/13877/files#diff-867a7da9528d469cff66dc89ce807d2f917b345e517667e731466fe1523433e6R58).